### PR TITLE
2024 Update

### PR DESCRIPTION
**Pull request by cromewar**
- Added all the links to point to updraft for all sections.
- Changed the video thumbnails and links to point to youtube.
- Added new sections for the update. (wallet attraction and more).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,30 +49,14 @@ Please refer to this for an in-depth explanation of the content:
...
-      <a href="https://youtu.be/sas02qSFZ74" target="_blank">
-        <img src="./thumbnails/thumbnail-2.png" width="300" alt="Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI 2">
-      </a>
-    </td>
-    <td align="center">
-      <a href="https://youtu.be/wUjYK5gwNZs" target="_blank">
-        <img src="./thumbnails/thumbnail-3.png" width="300" alt="Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI 3">
+
+<p align="center">
+<a href="https://youtu.be/WIsXSwo4iDc?si=PkLPgHmsmlHE07wF" target="_blank">
```
Comment (PatrickAlphaC) [inline README.md:54]:
```suggestion
<a href="https://updraft.cyfrin.io/courses/blockchain-basics" target="_blank">
```

File: README.md [hunk 2]
```diff
@@ -49,30 +49,14 @@ Please refer to this for an in-depth explanation of the content:
...
+        <img src="./thumbnails/2024-update.png" width="750" alt="Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI 1">
```
Comment (PatrickAlphaC) [inline README.md:55]:
```suggestion
        <img src="./thumbnails/2024-update.png" width="750" alt="Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI">
```

File: README.md [hunk 3]
```diff
@@ -82,9 +66,7 @@ Cyfrin Updraft Courses:
 - [Cyfrin Updraft - Foundry Fundamentals](https://updraft.cyfrin.io/courses/foundry)
 - [Cyfrin Updraft - Advanced Foundry](https://updraft.cyfrin.io/courses/advanced-foundry)
 
-[YouTube Playlist](https://www.youtube.com/playlist?list=PL4Rj_WH6yLgWe7TxankiqkrkVKXIwOP42)
-
-[YouTube Playlist of each section](https://www.youtube.com/playlist?list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk)
+[YouTube Link](https://youtu.be/WIsXSwo4iDc?si=PkLPgHmsmlHE07wF)
```
Comment (PatrickAlphaC) [inline README.md:69]:
```suggestion
[YouTube Link](https://www.youtube.com/watch?v=-1GB6m39-rM)
```

File: README.md [hunk 4]
```diff
@@ -826,20 +803,23 @@ _[⌨️ (1:51:36) | High-Level Blockchain Fundamentals](https://www.youtube.com
...
 <p align="right">(<a href="#table-of-contents">back to top</a>) ⬆️</p>
 
-# Section 2: Welcome to Remix - Remix Simple Storage
+# Course: Solidity & Smart Contract Developmnet
+
+[⭐️ Learn Solidity & Smart Contract Development](https://updraft.cyfrin.io/courses/solidity)
 
 💻 Code: [https://github.com/cyfrin/remix-simple-storage-cu](https://github.com/cyfrin/remix-simple-storage-cu)
 
+
```
Comment (PatrickAlphaC) [inline README.md:812]:
```suggestion
## Solidity 101 Section 1: Simple Storage
```

File: README.md [hunk 5]
```diff
@@ -919,81 +899,83 @@ _[⌨️ (3:17:46) | Deploying your First Contract](https://youtu.be/umepbfKp5rI
...
 ## zkSync L2 deploy
 
+_[⌨️ Deploying to Zksync](https://updraft.cyfrin.io/courses/solidity/simple-storage/zksync-deploying)_
+
 - zkSync compiler edition: 0.8.24
 
 ## 🐸🐦 [Tweet Me (add your contract in)!](https://twitter.com/intent/tweet?text=I%20just%20deployed%20a%20Smart%20Contract%20to%20the%20@zksyncdevs%20testnet!%0a%0a%20Thanks%20@PatrickAlphaC%20and%20@cyfrinupdraft!)
 
-## The EVM & A Recap of Section 2
+## The EVM & A Recap of the course
```
Comment (PatrickAlphaC) [inline README.md:908]:
```suggestion
## The EVM & A Recap of the section
```

File: README.md [hunk 6]
```diff
@@ -919,81 +899,83 @@ _[⌨️ (3:17:46) | Deploying your First Contract](https://youtu.be/umepbfKp5rI
...
 
-_[⌨️ (3:27:07) | The EVM & A Recap of Section 2](https://youtu.be/umepbfKp5rI?t=12427)_
+_[⌨️ The EVM & A Recap of Section 2](https://updraft.cyfrin.io/courses/solidity/simple-storage/evm-recap)_
 
 - The EVM
 
-## Section 2 NFTs
+## Course Section 1 NFTs
```
Comment (PatrickAlphaC) [inline README.md:914]:
```suggestion
## Simple Storage NFTs
```

File: README.md [hunk 7]
```diff
@@ -919,81 +899,83 @@ _[⌨️ (3:17:46) | Deploying your First Contract](https://youtu.be/umepbfKp5rI
...
 
 - [zkSync](https://explorer.zksync.io/address/0x5f9f9f46ECb48D625e320888922Bf86555608E9C)
 - [Sepolia](https://sepolia.etherscan.io/address/0x5c1ddb86F11BB46D3067C702AC554aEaED9ff8f0#code)
 
 <p align="right">(<a href="#table-of-contents">back to top</a>) ⬆️</p>
 
-# Section 3: Remix Storage Factory
+# Course Section 2 | Remix Storage Factory
```
Comment (PatrickAlphaC) [inline README.md:921]:
```suggestion
# Solidity 101 Section 2 | Remix Storage Factory
```

File: README.md [hunk 8]
```diff
@@ -826,20 +803,23 @@ _[⌨️ (1:51:36) | High-Level Blockchain Fundamentals](https://www.youtube.com
 
 <p align="right">(<a href="#table-of-contents">back to top</a>) ⬆️</p>
 
-# Section 2: Welcome to Remix - Remix Simple Storage
+# Course: Solidity & Smart Contract Developmnet
```
Comment (PatrickAlphaC) [inline README.md:806]:
```suggestion
# Course: Solidity 101
```

File: README.md [hunk 9]
```diff
@@ -919,81 +899,83 @@ _[⌨️ (3:17:46) | Deploying your First Contract](https://youtu.be/umepbfKp5rI
...
 - [Override & Virtual Keyword](https://docs.soliditylang.org/en/latest/contracts.html?highlight=override#function-overriding)
 
-## Section 3 Recap
+## Section 2 Recap
 
-_[⌨️ (4:07:37) | Section 3 Recap](https://youtu.be/umepbfKp5rI?t=14857)_
+_[⌨️ Section 2 Recap](https://updraft.cyfrin.io/courses/solidity/storage-factory/summary-and-recap)_
 
-## Section 3 NFTs
+## Course Section 2 NFTs
```
Comment (PatrickAlphaC) [inline README.md:957]:
```suggestion
## Section 2 NFTs
```

File: README.md [hunk 10]
```diff
@@ -919,81 +899,83 @@ _[⌨️ (3:17:46) | Deploying your First Contract](https://youtu.be/umepbfKp5rI
...
 
 - [zkSync](https://explorer.zksync.io/address/0x1D5c2Eb5F17Fa0BA4E3DfDDBF59DC0e6316219Ff)
 - [Sepolia](https://sepolia.etherscan.io/address/0x4A62A8901e6113dEF0AEeEC77E531779cd40da73#code)
 
 <p align="right">(<a href="#table-of-contents">back to top</a>) ⬆️</p>
 
-# Section 4: Remix Fund Me
+# Section 3: Remix Fund Me
```
Comment (PatrickAlphaC) [inline README.md:964]:
```suggestion
# Solidity 101 Section 3: Remix Fund Me
```

File: README.md [hunk 11]
```diff
@@ -1138,34 +1116,34 @@ _[⌨️ (5:34:53) | Immutable & Constant](https://youtu.be/umepbfKp5rI?t=20093)
...
 - [Receive](https://docs.soliditylang.org/en/latest/contracts.html?highlight=fallback#receive-ether-function)
 
-## Section 4 Recap
+## Section 3 Recap
 
-_[⌨️ (5:57:10) | Section 4 Recap](https://youtu.be/umepbfKp5rI?t=21430)_
+_[⌨️ Section 3 Recap](https://updraft.cyfrin.io/courses/solidity/fund-me/recap-congratulations-fundme)_
 
-## Section 4 NFTs
+## Course Section 3 NFTs
```
Comment (PatrickAlphaC) [inline README.md:1135]:
```suggestion
## Section 3 NFTs
```

File: README.md [hunk 12]
```diff
@@ -1138,34 +1116,34 @@ _[⌨️ (5:34:53) | Immutable & Constant](https://youtu.be/umepbfKp5rI?t=20093)
...
 
 <p align="right">(<a href="#table-of-contents">back to top</a>) ⬆️</p>
 
-# Section 5: AI Prompting, Asking Questions, and Getting Help
+# Section 4: AI Prompting, Asking Questions, and Getting Help
 
 DO NOT SKIP THIS!!
 
-_[⭐️ (6:00:00) | Section 5 | AI Prompting](https://www.youtube.com/watch?v=umepbfKp5rI&t=21600s)_
+_[⭐️ Section 4 | AI Prompting](https://updraft.cyfrin.io/courses/solidity/ai-prompting/ai-and-forums)_
```
Comment (PatrickAlphaC) [inline README.md:1146]:
```suggestion
# Solidity 101 Section 4: AI Prompting

_[⭐️ Section 4 | AI Prompting](https://updraft.cyfrin.io/courses/solidity/ai-prompting/ai-and-forums)_
```

File: README.md [hunk 13]
```diff
@@ -1177,40 +1155,42 @@ _[⭐️ (6:00:00) | Section 5 | AI Prompting](https://www.youtube.com/watch?v=u
...
 
 ## SpeedRunEthereum
 
-_[⌨️ (6:19:58) | SpeedRunEthereum](https://youtu.be/umepbfKp5rI?t=22798)_
+_[⌨️ SpeedRunEthereum](https://updraft.cyfrin.io/courses/solidity/ai-prompting/speedrun)_
 
 - Special Guest [Austin](https://twitter.com/austingriffith)
 
-## Section 5 NFTs
+## Course Section 4 NFTs
```
Comment (PatrickAlphaC) [inline README.md:1174]:
```suggestion
## Section 4 NFTs
```

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
I don't understand the markdown hierarchy. Why is does `foundry fundamentals` have 2 `#`s, but a lot of the sections "inside" the course also have 2 `#`s?

```
## Foundry fundamentals
```

Can we have a standard?

Course layout:
```
# <course name>
## <course name> | <section name>
### <lesson>
#### <additional stuff>
```

can we also not have it say `course NFT`, it doesn't make sense. it's the `section` nft.

Remember, the hierarchy is:

```
course -> section -> lesson
```

File: README.md (hunk 6 above)
Comment (jefrenchee) [reply README.md]: tank you

**Pull request merged**